modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -531,26 +590,38 @@ private void defineConnectorInitFunction(BLangConnector connector) {
         BLangUserDefinedType connectorType = (BLangUserDefinedType) TreeBuilder.createUserDefinedTypeNode();
         connectorType.pos = connector.pos;
         connectorType.typeName = connector.name;
+        connectorType.pkgAlias = (BLangIdentifier) TreeBuilder.createIdentifierNode();
```

Comment:
This should happen at the BLangPackageBuilder level

Reply:
I'm not clear on this, how do we do that in package builder level?